requirements.txt

```diff
@@ -4,3 +4,4 @@ watchdog==2.2.1
 multiprocess==0.70.14
 jinja2==3.1.2
 nestd==0.3.1
+inquirerpy==0.3.4
```

Comment:
One question - did you have a look at this library? - https://github.com/CITGuru/PyInquirer

Reply:
Hey 👋 @sansyrox You are looking for a wrong inquirerpy repository. I'm using https://github.com/kazhala/InquirerPy this one. Pypi - https://pypi.org/project/inquirerpy/

Reply:
Sure I'll have a look on this repo. btw I'm using this one https://pypi.org/project/inquirerpy/. Should I migrate it to the one you are saying ?

Reply:
Hey @Mr-Sunglasses 👋 

Apologies for this late revert. This message missed my eye 😅  

I am not very familiar with either of the repos but `inquirepy` seems more active than `PyInquirer`. So, we can use `inquirepy` for now. :D